app/services/delete_team.rb

```diff
@@ -0,0 +1,109 @@
+class DeleteTeam
+  include Interactor
+
+  delegate :team, :new_team, :user, to: :context
```

Comment:
Suggest rewording this to be clearer that it’s deleting a team and merging all its users & cases into another team:

```suggestion
class MergeTeams
  include Interactor

  delegate :team_to_delete, :team_to_keep, :user, to: :context
```

Reply:
🤔 not sure about this. The merging is the thing you have to do to accomplish the team deletion.